### DISCUSSION TITLE
Should we parse `runtime/doc/deprecated.txt` for deprecated options in Neovim?

### DISCUSSION THREAD

**Discussion by lambdalisue** [vim-denops/denops-std-deno#231] [Ideas]
For example, `paste` option in Neovim has deprecated and the definition has moved into `runtime/doc/deprecated.txt` that why `paste` option is not listed in `option` module

https://github.com/neovim/neovim/commit/2daf0b37dbfe54a4510c1033531dbaefd168c387

We are not sure if we should parse the file while the format is quite different.

**Comment by Shougo** [comment]
I think it should be parsed.  Because it is not deprecated in Vim.

**Comment by Shougo** [comment]
Oh, `paste` option is included in https://deno.land/x/denops_std@v6.0.1/option/vim/mod.ts

**Reply by lambdalisue** [reply]
Yes. That's why it is **correct** for denops_std, at least for now.

**Reply by Shougo** [reply]
OK.  So I don't need the feature.
